Cabal-syntax/src/Distribution/PackageDescription/FieldGrammar.hs

```diff
@@ -621,6 +621,7 @@ buildInfoFieldGrammar =
     <*> monoidalFieldAla "cc-options" (alaList' NoCommaFSep Token') L.ccOptions
     <*> monoidalFieldAla "cxx-options" (alaList' NoCommaFSep Token') L.cxxOptions
       ^^^ availableSince CabalSpecV2_2 []
+    <*> monoidalFieldAla "js-options" (alaList' NoCommaFSep Token') L.jsOptions
```

Comment:
See above, needs to be guarded by `availableSince`.

Reply:
I thought about it but got confused by a fact `js-sources` are not guarded. Is it an omission?

Which value should I put here?

Reply:
@mpickering ?